ui/Screen/Project.svelte

```diff
@@ -87,7 +87,7 @@
 
   let registerProjectMenuItem;
 
-  if (session.identity.registered) {
+  if (session.permissions.registerProject && session.identity.registered) {
```

Comment:
This logic should be behind the bool returned from the proxy, it should only be true if the project can be registered and the session has a registered user handle. I assume this will come once we have permissions per entity?

Reply:
Looking at the proxy part this condition is alreayd checked, so we can drop the identity check here.

Reply:
removed in 8092bbc